encryption_works.md

```diff
@@ -96,6 +96,32 @@ It's important to remember that just because you use free software doesn't mean
 
 Tails, which is a live DVD and live USB GNU/Linux distribution that I will discuss in detail below, solves many of these problems.
 
+## Signal and TextSecure
+
+> Now we have free, easy-to-use tools which you can download on your smartphone, which is sitting in the room with you. For the iPhone, there's a program called Signal, by Open Whisper Systems. It's very good, I know the security model. \
+— Edward Snowden on protecting communications in transit. ([Source](https://www.youtube.com/watch?t=149&v=j_kieJ-Ng2Q).)
```

Comment:
Can we copy the formatting used by the first quote from Snowden at the top of the document? (starts with "Encryption works"). That formatting looks good to me, this not so much.